cli/src/main.rs

```diff
@@ -149,13 +149,23 @@ impl CompileArgs {
     }
 }
 
+#[derive(Parser)]
+struct InitArgs {
+    #[arg(
+        index = 1,
+        help = "What language the template will be made in. Currently available: c++, c and rust."
```

Comment:
You can probably add "list" as a special *language* which just lists all available languages. Regardless if you do it this way or with a flag, I think hard-coding is bad since then we need to keep it in sync with templates.json, and if we move that to a separate repo, we would need to change ModMark to facilitate adding new templates.

```suggestion
        help = "What language the template will be made in. Enter 'list' for a list of languages."
```